crates/core/src/ring_switch/prove.rs

```diff
@@ -291,7 +304,13 @@ fn make_ring_switch_eq_ind<P, Tower>(
 	mixing_coeff: FExt<Tower>,
 ) -> Result<MultilinearWitness<'static, P>, Error>
 where
-	P: PackedFieldIndexable<Scalar = FExt<Tower>>,
+	P: PackedFieldIndexable<Scalar = FExt<Tower>>
```

Comment:
This can probably be `P: PackedFieldIndexable<Scalar = FExt<Tower>> + PackedTop<Tower>`